lib/libc/gen/tls_malloc.c

```diff
@@ -92,7 +98,9 @@ static void *__tls_malloc_aligned(size_t size, size_t align);
  * byte is set to MAGIC, and the second byte is the size index.
  */
 union	overhead {
+	struct {
```

Comment:
This is odd

Reply:
Indeed. That looks like a pointless leftover of some sort.